lib/controllers/orders.js

```diff
@@ -4,59 +4,44 @@ const pool = require('../utils/pool');
 
 module.exports = Router()
   .post('/', async (req, res) => {
-    const { rows } = await pool.query(
-      'INSERT INTO orders(product, quantity) VALUES ($1, $2) RETURNING *;',
-      [req.body.product, req.body.quantity]
-    );
-    const order = new Order(rows[0]);
+    const resp = await Order.insert({
+      product: req.body.product,
+      quantity: req.body.quantity,
+    });
 
-    res.json(order);
+    res.json(resp);
   })
 
-  .get('/:id', async (req, res) => {
-    const { rows } = await pool.query('SELECT * FROM orders WHERE id=$1;', [
-      req.params.id,
-    ]);
-
-    if (!rows[0]) return null;
-    const order = new Order(rows[0]);
-
-    res.json(order);
+  .get('/:id', async (req, res, next) => {
+    try {
+      const order = await Order.getById(req.params.id);
+      res.send(order);
+    } catch (error) {
+      error.status = 404;
```

Comment:
if you're using next, a cool middleware feature, you don't need to include any error.status. 
without next you'd use something like `res.status(500).send(error)`
otherwise, in the catch block, if using next, just use. `next(error)`